controllers/apps/cluster_controller_test.go

```diff
@@ -107,7 +107,7 @@ var _ = Describe("Cluster Controller", func() {
 		testapps.ClearResourcesWithRemoveFinalizerOption(&testCtx, generics.BackupSignature, true, inNS, ml)
 		testapps.ClearResourcesWithRemoveFinalizerOption(&testCtx, generics.BackupPolicySignature, true, inNS, ml)
 		testapps.ClearResourcesWithRemoveFinalizerOption(&testCtx, generics.VolumeSnapshotSignature, true, inNS)
```

Comment:
Add `ml` param.

Reply:
I think the service named `kubernetes` is a special case. Other resources are not likely to suffer from this problem.
Plus I searched a suffix and there are too many resources that are cleaned without specifying the `TestObjLabelKey` label. Changing them all seems not necessary.
<img width="273" alt="image" src="https://github.com/user-attachments/assets/fdbf1d63-ed6e-4013-99a1-c4eb5ae2ec30">